client/gutenberg/extensions/shortlinks/editor.scss

```diff
@@ -0,0 +1,9 @@
+.jetpack-shortlinks__panel {
+	.components-base-control {
+		display: inline-block;
+	}
+
+	.components-clipboard-button {
+		margin: 1px 0 0 2px;
```

Comment:
Can you change this to `margin-top: 1px;`? Then, can you add a `margin-right: 4px;` to the text input? This way it will wrap nicely on teeny screens.

Reply:
Good idea 👍 Done in https://github.com/Automattic/wp-calypso/pull/29556/commits/1c05b9d7366cd70ac6b834f0d65c4312d46e41a4 - thanks!